exercise/staff_views.py

```diff
@@ -530,3 +532,61 @@ def form_valid(self, form):
     def form_invalid(self, form):
         messages.error(self.request, _('FAILURE_SAVING_CHANGES'))
         return super().form_invalid(form)
+
+class SubmissionConversion(SubmissionMixin, BaseRedirectView):
+    """
+    A POST-only view that updates a student's late or unofficial submission
+    to normal submission. Changed the status and remove the penalty
+    """
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
```

Comment:
Is this missing access_mode = ACCESS.ASSISTANT?

Reply:
Well, i'm not sure, but if you think i should add that then i could do that